torch/_inductor/scheduler.py

```diff
@@ -894,6 +875,12 @@ def can_fuse(self, node1: BaseSchedulerNode, node2: BaseSchedulerNode):
             return False  # node2 must go before node1
         if node2.is_template():
             return False  # only epilogues
+        if node1.is_template() and (
+            node2.has_aliasing_or_mutation()
```

Comment:
why do we want to skip if there is aliasing ?

Reply:
We may want to support this eventually, though for now being conservative of when to apply this.